datashuttle/utils/validation.py

```diff
@@ -47,6 +49,13 @@ def validate_list_of_names(
 
     log: bool
         If `True`, output will also be logged to "datashuttle" logger.
+
+    name_templates : Dict
+        e.g. {"name_templates": {"on": False, "sub": None, "ses": None}}
```

Comment:
being pedantic here. It's better for the order of parameters in the docstring to match that in the functions signature (`log` comes before `name_templates` in docstring, but other way around in the signature).